src/theme/components/pagination.ts

```diff
@@ -9,11 +9,6 @@ const Pagination = {
       lineHeight: "1.15",
       textDecoration: "none",
     },
-    li: {
-      _first: {
-        marginTop: "xxs",
-      },
-    },
   },
```

Comment:
Add styles to remove the default bottom margin on the internal `ul` element.

Reply:
Thanks @bigfishdesign13, let me know if these updates are what you had in mind!